File: ios/DatalakeFaceAuth/FaceCaptureViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(FaceCaptureViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(onFaceProcessed, RCTDirectEventBlock)
RCT_EXPORT_VIEW_PROPERTY(onLivenessUpdate, RCTDirectEventBlock)
RCT_EXPORT_VIEW_PROPERTY(onStatusMessage, RCTDirectEventBlock)

@end

File: ios/DatalakeFaceAuth/FaceCaptureViewManager.swift
import Foundation
import React

@objc(FaceCaptureViewManager)
class FaceCaptureViewManager: RCTViewManager {
    
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    override func view() -> UIView! {
        return FaceCaptureView()
    }
}
